sql/core/src/main/scala/org/apache/spark/sql/execution/command/variables.scala

```diff
@@ -119,13 +116,45 @@ case class DropVariableCommand(
 /**
  * This command is for setting a SQL variable.
  */
-case class SetVariableCommand(identifier: VariableIdentifier,
-                              newExpr: String) extends LeafRunnableCommand {
+case class SetVariableCommand(identifierList: Seq[VariableIdentifier],
```

Comment:
nit: `identifiers` plural 🙏

Reply:
Not a native speaker, but I think singular is correct here: List Of Identifiers but identifierList.